### ISSUE TITLE
UI/UX Improvement to see more builds in less space on /builds page

### ISSUE THREAD

**Issue by aaronorosen** [buildkite/feedback#404]
It would be nice is there was a trimmed down view of this page so that I could see more information about the pipeline in way less space: 

![image](https://user-images.githubusercontent.com/1063157/41206969-e4ccd876-6cc1-11e8-8021-cf53a5a7bd01.png)

**Comment by petemounce** [comment]
Yes, agree. Scrolling on a mobile phone requires quite a lot of calories ;)
